gobblin-hive-registration/src/main/java/gobblin/hive/HiveRegProps.java

```diff
@@ -54,6 +54,8 @@
   private final State storageProps;
   private final State serdeProps;
 
+  private Optional<String> runtimeExtraTableProps;
```

Comment:
There's already tableProps exists which represent the static table properties obtained from .job file, if any. 

Here there are some table props obtained from runtime. May be `runtimeTableProps` is better?